wireless_msgs/msg/Connection.msg

```diff
@@ -11,3 +11,4 @@ int16 noise_level  # dBm
 
 string essid
 string bssid
+float64 frequency
```

Comment:
Is this necessary? What about the frequency as reported in Network? Can that be cross-referenced using essid, or is that just way too much trouble?